dev.omerkurt.Tureng.json

```diff
@@ -0,0 +1,55 @@
+{
+  "app-id": "dev.omerkurt.Tureng",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "turengvocabulary",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--socket=pulseaudio"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    "python3-modules.json",
+    {
+      "name": "blueprint",
+      "cleanup": [
+        "*"
+      ],
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+          "commit": "2a39a16391122af2f3d812e478c1c1398c98b972"
+        }
+      ]
+    },
+    {
+      "name": "Tureng",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "tag": "v1.0.0",
+          "url": "https://github.com/omerkurtdev/dev.omerkurt.Tureng.git"
```

Comment:
Retagging is bad.

please add the `commit` as well to protect against it.

Reply:
I'm having a problem with the application icon, I have a few more things to edit, I'm sorry, I won't tag it again anymore, I'll add a direct commit.

https://github.com/omerkurtdev/dev.omerkurt.Tureng/commit/7c212b9e5f8b3749086e2fdb3db0ea74a2ef8cf1#diff-9790b2ef4d3a8a0757749534f76413e015621d1ad3bac24261f7d33e89785b57L5